share/smack/include/smack.di

```diff
@@ -0,0 +1,2 @@
+
+extern (C): void __VERIFIER_assert(int);
```

Comment:
What about `__VERIFIER_nondet_int` and such?

Reply:
The only method required for the benchmarks I've been using so far is `__VERIFIER_assert`. I can add those method signatures here, but they will be untested and may or may not work.